test/rules.json

```diff
@@ -1339,6 +1339,54 @@
       "transformers": [
         "lowercase"
       ]
+    },
+    {
+      "id": "test-marshalling",
```

Comment:
keep this new rule above the "long rule", as it's meant to be the last rule on purpose because it's so long it's annoying to scroll past it